### PR TITLE
Pr164- Identityes created by Qin Wu

### PR DESCRIPTION
**Pull request by sbarguil**

### CODE REVIEW

File: yang/IETF-04/ietf-vpn-common.yang [hunk 1]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
     base protocol-type;
     description
       "Transport based on SR.";
-    reference
-      "";
+      reference 
+      "RFC8660:Segment Routing with MPLS data plane 
+       RFC8663:SR-MPLS over IP
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:580]:
```suggestion
       RFC 8663: MPLS Segment Routing over IP 
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 2]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
     base protocol-type;
     description
       "Transport based on SR.";
-    reference
-      "";
+      reference 
+      "RFC8660:Segment Routing with MPLS data plane 
+       RFC8663:SR-MPLS over IP  
+       RFC8754:IPv6 Segment Routing Header (SRH)";
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:581]:
```suggestion
       RFC 8754: IPv6 Segment Routing Header (SRH)";
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 3]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
...
   }
 
   identity sr-te {
     base protocol-type;
     description
       "Transport based on SR-TE.";
-    reference
-      "";
+    reference 
+     "RFC8426:Recommendations for RSVP-TE and Segment Routing (SR)
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:589]:
```suggestion
     "RFC 8426: Recommendations for RSVP-TE and Segment Routing (SR)
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 4]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
...
+      Label Switched Path (LSP) Coexistence";
   }
 
   identity rsvp-te {
     base protocol-type;
     description
       "Transport based on RSVP-TE.";
     reference
-      "";
+      "RFC2205: Resource ReSerVation Protocol (RSVP) --
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:598]:
```suggestion
      "RFC 2205: Resource ReSerVation Protocol (RSVP) -- 
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 5]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
     base protocol-type;
     description
       "Transport based on SR.";
-    reference
-      "";
+      reference 
+      "RFC8660:Segment Routing with MPLS data plane
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:579]:
```suggestion
      "RFC 8660: Segment Routing with the MPLS Data Plane 
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 6]
```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
     base protocol-type;
     description
       "Transport based on SR.";
-    reference
-      "";
+      reference
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:578]: We will need to add all these references to the Informative References of the draft and call them in the description text.

File: yang/IETF-04/ietf-vpn-common.yang [hunk 7]
```diff
@@ -1070,4 +1074,24 @@ module ietf-vpn-common {
       }
     }
   }
+
+   grouping vpn-node-group {
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:1078]: Please use 2-space indent. thanks

Comment (sbarguil) [reply yang/IETF-04/ietf-vpn-common.yang:1078]: Oki :)

File: yang/IETF-04/ietf-vpn-common.yang (hunk 6 above)
Comment (sbarguil) [reply yang/IETF-04/ietf-vpn-common.yang:578]: Updated

**Pull request merged**